SurfaceToolbox/SurfaceToolbox.py

```diff
@@ -406,7 +406,10 @@ def remesh(inputModel, outputModel, subdivide=0, clusters=10000):
 
     import pyacvd
     import pyvista as pv
-    inputMesh = pv.wrap(inputModel.GetPolyData())
+    tri_filter = pv._vtk.vtkTriangleFilter()

```

Comment:
Was there any specific reason for getting VTK from pyvista? If there is no specific reason then let's just use `vtk` as everywhere else in the file.
```suggestion
    tri_filter = vtk.vtkTriangleFilter()

```

Reply:
I agree, there is no reason to use VTK from PyVista.